agent/tool-scripts/pprof

```diff
@@ -105,79 +104,37 @@ tool_cmd_file="$tool_output_dir/$tool.cmd"
 tool_output_file=$tool_output_dir/$tool.txt
 export PPROF_TMPDIR=$tool_output_dir
+declare -a hosts=($(cat $inventory | sed -n '/^\[pprof\]/,/^$/p' | awk 'length {if (NR!=1) {print}}'))
 
 # function to setup openshift master or node file to support web profile, depending
 # is it Openshift Enterprise, or upstream Openshift Origin
-ose_pprof() {
-    case "$osecomponent" in
-        master)
-            if [ -e $origin-master ]; then 
-                if grep -q "^OPENSHIFT_PROFILE=web" $origin_master; then
-                    systemctl restart openshift-master.service
-                else
-                    echo "OPENSHIFT_PROFILE=web" >> $origin_master
-                    systemctl restart openshift-master.service
-                fi
-            else
-                #check if openshift-master service is master, that means its a ha installation
-                if [ $(systemctl list-unit-files | grep atomic-openshift-master.service | awk {'print $2'}) == "masked" ]; then
-                    if grep -q "^OPENSHIFT_PROFILE=web" $ose_openshift_master_api; then
-                        systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
-                    else
-                        echo "OPENSHIFT_PROFILE=web" >> $ose_openshift_master_api
-                        echo "OPENSHIFT_PROFILE=web" >> $ose_openshift_master_controllers
-                        echo "OPENSHIFT_PROFILE_PORT=6061" >> $ose_openshift_master_api
-                        systemctl restart atomic-openshift-master-api.service atomic-openshift-master-controllers.service
-                    fi
-                else
-                    if grep -q "^OPENSHIFT_PROFILE=web" $ose_openshift_master; then
-                        systemctl restart atomic-openshift-master
-                    else
-                        echo "OPENSHIFT_PROFILE=web" >> $ose_openshift_master
-                        systemctl restart atomic-openshift-master.service
-                    fi
-                fi
-            fi 
-            ;;
-        node)
-            if [ -e $origin-node ]; then 
-                if grep -q "^OPENSHIFT_PROFILE=web" $origin_node; then
-                    systemctl restart openshift-node.service
-                else
-                    echo "OPENSHIFT_PROFILE=web" >> $origin_node
-                    systemctl restart openshift-node.service
-                fi 
-            else
-                if grep -q "^OPENSHIFT_PROFILE=web" $ose_openshift_node; then
-                    systemctl restart atomic-openshift-node.service
-                else
-                    echo "OPENSHIFT_PROFILE=web" >> $ose_openshift_node
-                    systemctl restart atomic-openshift-node.service
-                fi
-            fi 
-    esac
-}
 
 collect_data() {
     while true; do
         if [ $(systemctl list-unit-files | grep atomic-openshift-master.service | awk {'print $2'}) == "masked" ]; then
```

Comment:
containerized environments won't have access to systemd unit files, so I guess we should have an ssh call to the host to find out if it's a master.